bin/build_roct.sh

```diff
@@ -65,13 +65,12 @@ if [ "$1" == "install" ] ; then
    $SUDO rm $INSTALL_ROCT/testfile
 fi
 
+patchfile=$thisdir/patches/roct_ppc.patch
+patchdir=$AOMP_REPOS/$AOMP_ROCT_REPO_NAME
+patchrepo
```

Comment:
I am not sure I agree with moving the patch logic outside of the condition below because this patch will apply on both build of roct and install of roct.

Reply:
This is ok to move patching to both install and build.